src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -70,12 +70,25 @@ LegacyConfigForm.propTypes = {
     allowAnonymousPosts: PropTypes.bool.isRequired,
     allowAnonymousPostsPeers: PropTypes.bool.isRequired,
     blackoutDates: PropTypes.string.isRequired,
-  }).isRequired,
+  }),
   onSubmit: PropTypes.func.isRequired,
   // eslint-disable-next-line react/forbid-prop-types
   formRef: PropTypes.object.isRequired,
   intl: intlShape.isRequired,
   title: PropTypes.string.isRequired,
 };
 
+LegacyConfigForm.defaultProps = {
```

Comment:
You'll see a few of these - we're making use of `defaultProps` to create the `appConfig` shape in the event that we don't have a config for this provider already (in the case that we're switching providers, basically)